sympy/stats/joint_rv_types.py

```diff
@@ -717,3 +719,73 @@ def NegativeMultinomial(syms, k0, *p):
     if not isinstance(p[0], list):
         p = (list(p), )
     return multivariate_rv(NegativeMultinomialDistribution, syms, k0, p[0])
+
+
+# -------------------------------------------------------------------------------
+# Wishart distribution ---------------------------------------------------------
+
+class WishartDistribution(JointDistribution):
+    _argnames = ['n', 'V']
+
+    is_Continuous = True
+
+    @property
+    def set(self):
+        raise NotImplementedError()
+
+    @staticmethod
+    def check(n, V):
+        p = V.shape[0]
+        _value_check(n > p - 1,
+                     "Degrees of freedom must be greater than p")
+        # check if scale matrix is positive definite or not.
+        if not isinstance(V, MatrixSymbol):
+            _value_check(V.is_positive_definite,
+                         "The scale matrix must be positive definite. ")
+
+    def pdf(self, args):
+        n, V = self.n, self.V
+        p = V.shape[0]
+        if isinstance(args, MatrixSymbol):
+            x = args
+        else:
+            x = ImmutableMatrix(args)
+        num = det(x) ** (n - p - 1) * exp(-Trace(Inverse(V) * x) / 2)
+        den = 2 ** (n * p / 2) * det(V) ** (n / 2) * multigamma(n / 2, p)
+        return num / den
+
+
+def Wishart(name, n, V):
+    """
+    Creates a joint random variable with Wishart distribution.
+
+    Parameters
+    ==========
+
+    name: string, representing name of Wishart distribution
+    n: int, degrees of freedom, Real
+    V: The shape matrix for the distribution, positive definite
+
+
+    Returns
+    =======
+
+    A random symbol
+
+
+    Examples
+    ========
+    >>> from sympy.stats import Wishart, density
+    >>> X = Wishart('x', 2, [[2, 1], [1, 2]])
```

Comment:
Can we support this case also, `Wishart('x', 2, Matrix([[2, 1], [1, 2]]))` in the API. May be is user wants to directly pass the matrix rather than as a nested list?

Reply:
This is supported

Reply:
Feel free to resolve this then. May be adding an example in the docs using `Matrix` will be easy for the user to understand.